docs/extend/extend-cli/cli-extending-a-plugin.md

```diff
@@ -10,77 +11,85 @@ This tutorial demonstrates how to extend the plug-in that is bundled with this s
 
-Specifically, we're going to show data from [this URI](https://jsonplaceholder.typicode.com/todos) by [Typicode](https://jsonplaceholder.typicode.com/).
-Typicode serves sample REST JSON data for testing purposes.
+Specifically, the tutorial shows data **[is the phrasing here correct?]** from [this URI](https://jsonplaceholder.typicode.com/todos) by [Typicode](https://jsonplaceholder.typicode.com/). Typicode serves sample REST JSON data for testing purposes.
 
-At the end of this tutorial, you will be able to use a new command from the Zowe CLI interface: `zowe zowe-cli-sample list typicode-todos`
+At the end of this tutorial, you are able to use the following new command from the Zowe CLI interface:
 
-Completed source for this tutorial can be found on the `typicode-todos` branch of the zowe-cli-sample-plugin repository.
+```
+zowe zowe-cli-sample list typicode-todos
+```
+
+The completed source for this tutorial can be found on the `typicode-todos` branch of the [`zowe-cli-sample-plugin`](https://github.com/zowe/zowe-cli-sample-plugin/#zowe-cli-sample-plug-in) repository.
 
 ### Creating a Typescript interface for the Typicode response data
 
-First, we'll create a Typescript interface to map the response data from a server.
+First, create a Typescript interface to map the response data from a server:
 
-Within `zowe-cli-sample-plugin/src/api`, create a folder named `doc` to contain our interface (sometimes referred to as a "document" or "doc"). Within the doc folder, create a file named `ITodo.ts`.
+1. Within `zowe-cli-sample-plugin/src/api`, create a folder named `doc` to contain the interface (sometimes referred to as a "document" or "doc"). **[looking for clarity here: is the interface what is referred to as document or doc?]**
 
-The `ITodo.ts` file will contain the following:
+2. In the `doc` folder, create a file named `ITodo.ts`.
 
-```typescript
-export interface ITodo {
-    userId: number;
-    id: number;
-    title: string;
-    completed: boolean;
-}
-```
+3. Edit the `ITodo.ts` file to contain the following code: **[correct?]**
+
+    ```typescript
+    export interface ITodo {
+        userId: number;
+        id: number;
+        title: string;
+        completed: boolean;
+    }
+    ```
 
 ### Creating a programmatic API
 
-Next, we'll create a Node.js API that our command handler uses. This API can also be used in any Node.js application, because these Node.js APIs make use of REST APIs, Node.js APIs, other NPM packages, or custom logic to provide higher level functions than are served by any single API.
+Next, create a Node.js API for the command handler to use. **[correct?]**
 
-Adjacent to the existing file named `zowe-cli-sample-plugin/src/api/Files.ts`, create a file `Typicode.ts`.
+This API can also be used in any Node.js application, because these Node.js APIs make use of REST APIs, Node.js APIs, other NPM packages, or custom logic to provide higher level functions than are served by any single API.
```

Comment:
Maybe there should be a period after "This API can also be used in any Node.js application". I think the rest could be deleted. 

It's not clear to me what the point the rest of the sentence is trying to make or how it relates to the beginning of the sentence.